dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/metrics/WorkerServerMetrics.java

```diff
@@ -30,13 +30,13 @@ public WorkerServerMetrics() {
     }
 
     private static final Counter WORKER_OVERLOAD_COUNTER =
-            Counter.builder("dolphinscheduler_worker_overload_count")
-                    .description("worker load count")
+            Counter.builder("ds.system.worker.overload.count")
+                    .description("overloaded workers count")
                     .register(Metrics.globalRegistry);
 
     private static final Counter WORKER_SUBMIT_QUEUE_IS_FULL_COUNTER =
-            Counter.builder("dolphinscheduler_worker_submit_queue_is_full_count")
-                    .description("worker task submit queue is full count")
+            Counter.builder("ds.system.worker.full.submit.queue.count")
```

Comment:
How about remove the system?
```suggestion
            Counter.builder("ds.worker.full.submit.queue.count")
```

Reply:
Sure, will remove `system`, since it might cause some ambiguities with jvm / network level stuff.